js/packages/utils/permissions/handle-press-permissions/prepareMediaAndSend.ts

```diff
@@ -0,0 +1,50 @@
+import beapi from '@berty/api'
+
+import { PrepareMediaAndSendProps } from './interfaces'
+
+const amap = async <T extends any, C extends (value: T) => any>(arr: T[], cb: C) =>
+	Promise.all(arr.map(cb))
+
+export const prepareMediaAndSend: (props: PrepareMediaAndSendProps) => void = async ({
+	sending,
+	setSending,
+	messengerClient,
+	onClose,
+	res,
+}) => {
+	try {
+		if (sending) {
```

Comment:
this might be out of date

Reply:
What do you mean by out of date ?

Reply:
for example here: https://github.com/berty/berty/pull/4316/files/d4a1108ee45312079635f76887c02a6670915ae9#diff-a4fa0c07ae99f80ca0174f3eed339c9d3abb1588e397e5d012c7a517d5666aa1R32

by the time this part is run, the value of sending might be different